[dev] npm run clean and nmp run dist not working @ Windows (cmd)

npm run clean:

```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! asciidoctor-kroki@0.12.0 clean: `rm -rf dist/*`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the asciidoctor-kroki@0.12.0 clean script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/README.md
+++ b/README.md
@@ -422,25 +422,28 @@ asciidoctor.convertFile('file.adoc', { safe: 'safe', attributes: { 'kroki-server
 
 To build this project, you will need the latest active LTS of Node.js and npm (we recommend `nvm` to manage multiple active Node.js versions).
 
-The latest Node LTS version is : ```v14.15.x```
+The current latest Node LTS version is: `v14.15.x`
 
-Please use latest NPM version ```v7.x``` for ```"lockfileVersion": 2```, see [lockfileversion](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion)
+Please use latest npm version `v7.x` to generate lockfile using v2 format (i.e., `"lockfileVersion": 2`), see [lockfileversion](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion)
 
 #### Update NPM @ Linux
+
     npm i -g npm
 
 #### Update NPM @ Windows
-see [npm-windows-upgrade](https://github.com/felixrieseberg/npm-windows-upgrade)
-1. Open PowerShell and select ```Run as Administrator```
 
-2. Install ```npm-windows-upgrade```
+1. Open PowerShell as Administrator selecting `Run as Administrator`
+
+2. Install `npm-windows-upgrade`
 
         Set-ExecutionPolicy Unrestricted -Scope CurrentUser -Force
         npm install --global --production npm-windows-upgrade
 
-3. Upgrade NPM
+3. Upgrade npm
 
         npm-windows-upgrade
+
+Reference: [npm-windows-upgrade](https://github.com/felixrieseberg/npm-windows-upgrade)
 
 ### Building
 

--- a/README.md
+++ b/README.md
@@ -422,6 +422,26 @@ asciidoctor.convertFile('file.adoc', { safe: 'safe', attributes: { 'kroki-server
 
 To build this project, you will need the latest active LTS of Node.js and npm (we recommend `nvm` to manage multiple active Node.js versions).
 
+The latest Node LTS version is : ```v14.15.x```
+
+Please use latest NPM version ```v7.x``` for ```"lockfileVersion": 2```, see [lockfileversion](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion)
+
+#### Update NPM @ Linux
+    npm i -g npm
+
+#### Update NPM @ Windows
+see [npm-windows-upgrade](https://github.com/felixrieseberg/npm-windows-upgrade)
+1. Open PowerShell and select ```Run as Administrator```
+
+2. Install ```npm-windows-upgrade```
+
+        Set-ExecutionPolicy Unrestricted -Scope CurrentUser -Force
+        npm install --global --production npm-windows-upgrade
+
+3. Upgrade NPM
+
+        npm-windows-upgrade
+
 ### Building
 
 1. Install the dependencies:

--- a/package.json
+++ b/package.json
@@ -14,9 +14,9 @@
     "test:antora": "mocha test/antora/**.spec.js",
     "lint": "standard src/**.js test/**.js",
     "lint-fix": "npm run lint -- --fix",
-    "clean": "rm -rf dist/*",
+    "clean": "shx rm -rf dist/*",
     "dist": "npm run clean && npm run dist:browser",
-    "dist:browser": "mkdir -p dist/browser && browserify src/asciidoctor-kroki.js --exclude ./node-fs.js --exclude ./fetch.js --exclude ./antora-adapter.js --standalone AsciidoctorKroki -o dist/browser/asciidoctor-kroki.js"
+    "dist:browser": "shx mkdir -p dist/browser && browserify src/asciidoctor-kroki.js --exclude ./node-fs.js --exclude ./fetch.js --exclude ./antora-adapter.js --standalone AsciidoctorKroki -o dist/browser/asciidoctor-kroki.js"
   },
   "dependencies": {
     "json5": "^2.1.3",
@@ -39,6 +39,7 @@
     "mocha": "^8.0.1",
     "puppeteer": "^7.0.1",
     "rimraf": "^3.0.2",
+    "shx": "^0.3.3",
     "sinon": "^9.0.1",
     "standard": "^16.0.2"
   },

